### PR TITLE
Revise various README.mds and tutorials

### PR DESCRIPTION
**Pull request by Fei-Guo**
- Restruct main README
- Revise UnitedDeployment tutorial

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,18 +20,16 @@ Today, Kruise offers four workload controllers:
...
-Several [tutorials](./docs/tutorial/README.md) are provided to demonstrate how to use the workload controllers.
-A [video](https://www.youtube.com/watch?v=elB7reZ6eAQ) by [Lachlan Evenson](https://github.com/lachie83) is also provided for demonstrating the controllers.
+This [video](https://www.youtube.com/watch?v=elB7reZ6eAQ) demo by [Lachlan Evenson](https://github.com/lachie83) is great for new users.
 
 ## Getting started
 
 ### Check before installation
 
-You should check the cluster before installation via the command-line with either `curl` or `wget`.
+Kruise requires APIServer to enable features such as `MutatingAdmissionWebhook` admission webhook. You can check your cluster qualification
```
Comment (FillZpp) [inline README.md]: `MutatingAdmissionWebhook` and `ValidatingAdmissionWebhook`

File: README.md [hunk 2]
```diff
@@ -69,107 +67,20 @@ kubectl apply -f https://raw.githubusercontent.com/kruiseio/kruise/master/config
...
-  podManagementPolicy: Parallel  # allow parallel updates, works together with maxUnavailable
-  updateStrategy:
-    type: RollingUpdate
-    rollingUpdate:
-      # Do in-place update if possible, currently only image update is supported for in-place update
-      podUpdatePolicy: InPlaceIfPossible
-      # Allow parallel updates with max number of unavailable instances equals to 2
-      maxUnavailable: 2
-```
+### Enable specific controllers
```
Comment (FillZpp) [inline README.md]: How about `### Optional: Enable specific controllers`

File: README.md (hunk 1 above)
Comment (Fei-Guo) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (Fei-Guo) [reply README.md]: done

**Comment by wu8685** [comment]
LGTM

**Pull request merged**